kickstarter.Rmd

```diff
@@ -186,4 +190,41 @@ print(catHeatPlot)
 ## Notes
 > summarize will strip out anything that isnt in the group by, mutation, or in the summarize itself
 > so you need to reference rows in the summarize if you want to include them but they're not in group by or mutate
-> Has kickstarter peaked in its popularity as a crowdfunding platform?
\ No newline at end of file
+
+---
```

Comment:
Added a separation to clarify that this notes section goes with the previous plot.